src/asciidoc/docs/develop/useGit.adoc

```diff
@@ -93,10 +95,25 @@ You will need to use this key each time you clone, pull, or push to the origin r
 WARNING: After you close the window, the access key will be displayed only by name (for example, `Token 1`).
 You will not be able to retrieve it from the Koji site again.
 You can, however, revoke your token and generate a new one if you have forgotten it, or if you believe that someone unauthorized has discovered it.
++
+NOTE: Keep the dialog open for the next two steps.
+
+=== Obtaining Koji environment variables
+
+[NOTE]
+If your project does not have a backend, skip this section.
+
+If your project has a backend, you need the values of the `KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables to access the leaderboard database.
+
+. If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*. The dialog opens.
```

Comment:
```suggestion
. If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*.
```
Removed because it's redundant.